README.md

```diff
@@ -61,7 +63,10 @@ In order to be useful, Anon needs to be told what you want to do to each column
     {
       // Takes a UK format postcode (eg. W1W 8BE) and just keeps the outcode
       // (eg. W1W).
-      "name": "outcode"
+      "name": "outcode",
+      // what field in the json this action needs to be applied. If a field in
```

Comment:
This feels a bit out of place — it suddenly talks about JSON when above we've changed it all to talk about there being multiple formats. Worth breaking out into a little explanation about how CSVs will infer the column number whereas JSON needs the `jsonField` to tell it which one you wanted?